packages/http-server/src/__tests__/fixtures/content-server-example.oas3.yaml

```diff
@@ -0,0 +1,52 @@
+---
+openapi: 3.0.0
+components:
+  examples:
+    PetHtml:
+      value: <html><p>Pet Example</p></html>
+    PetText:
+      value: Pet Example
+  schemas:
+    Pet:
+      type: string
+      
+paths:
+  "/pet/html":
+    get:
+      responses:
+        '200':
+          description: ''
+          content:
+            text/html:
+              schema:
+                "$ref": "#/components/schemas/Pet"
+              examples:
+                text/html:
+                  "$ref": "#/components/examples/PetHtml"
+  "/pet/plain":
+    get:
+      responses:
+        '200':
+          description: ''
+          content:
+            text/plain:
+              schema:
+                "$ref": "#/components/schemas/Pet"
+              examples:
+                text/plain:
+                  "$ref": "#/components/examples/PetText"
+servers:
+- url: "{schema}://{host}/{basePath}"
+  variables:
+    schema:
+      default: http
+      enum:
+      - http
+      - https
+    host:
+      default: stoplight.io
+      enum:
+      - stoplight.io
+      - dev.stoplight.io
+    basePath:
```

Comment:
Remove this document, it's not required anymore

Reply:
Fixed in 2fea3a0